rulr breaks on v12

```
Run ./start.sh
+ node -v
+ export NODE_VERSION=v12.22.10
+ npm start
> typescript-runtime-type-benchmarks@1.0.0 start /home/runner/work/typescript-runtime-type-benchmarks/typescript-runtime-type-benchmarks
> ts-node index.ts
/home/runner/work/typescript-runtime-type-benchmarks/typescript-runtime-type-benchmarks/node_modules/rulr/dist/rulr.modern.js:1
import{BaseError as r}from"make-error";import t from"validator";import e from"atob";class n extends r{constructor(){super("expected integer")}}function s(r){return Number.isInteger(r)}function o(r){if(s(r))return r;throw new n}class u extends r{constructor(){super("expected negative integer")}}function c(r){return s(r)&&r<=0}function i(r){if(c(r))return r;throw new u}class f extends r{constructor(){super("expected number")}}function a(r){return"number"==typeof r&&!1===Number.isNaN(r)}function p(r){if(a(r))return r;throw new f}class d extends r{constructor(){super("expected negative number")}}function h(r){return a(r)&&r<=0}function l(r){if(h(r))return r;throw new d}class w extends r{constructor(){super("expected positive integer")}}function x(r){return s(r)&&r>=0}function g(r){if(x(r))return r;throw new w}class m extends r{constructor(){super("expected positive number")}}function v(r){return a(r)&&r>=0}function y(r){if(v(r))return r;throw new m}class b extends r{constructor(){super("expected string")}}functio...
^^^^^^
SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/runner/work/typescript-runtime-type-benchmarks/typescript-runtime-type-benchmarks/cases/rulr.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Module.m._compile (/home/runner/work/typescript-runtime-type-benchmarks/typescript-runtime-type-benchmarks/node_modules/ts-node/src/index.ts:1056:23)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! typescript-runtime-type-benchmarks@1.0.0 start: `ts-node index.ts`
npm ERR! Exit status 1
npm ERR! 
```

--- a/cases/index.ts
+++ b/cases/index.ts
@@ -10,7 +10,7 @@ import './marshal';
 import './mojotech-json-type-validation';
 import './myzod';
 import './purify-ts';
-// import './rulr';
+import './rulr';
 import './runtypes';
 import './simple-runtypes';
 import './superstruct';

